src/Driver/driver_configs.jl

```diff
@@ -622,12 +625,24 @@ function SingleStackConfiguration(
         meshwarp = meshwarp,
     )
 
+    if cutofforder_horz < polyorder_horz && cutofforder_vert < polyorder_vert
+        filter = CutoffFilter(
+            grid,
+            (cutofforder_horz + 1, cutofforder_horz + 1, cutofforder_vert + 1),
+        )
+    else
```

Comment:
would it be useful to add an else if for cases where we only filter horizontal or vertical?

Reply:
Yeah, that's a good point, we might want to carefully handle that in the SingleStackConfiguration.

Reply:
We had  a similar [conversation](https://github.com/CliMA/ClimateMachine.jl/pull/1953#issuecomment-773493460) in PR #1953 . I put the `&&` in that condition, because a different condition for horizontal/vertical is currently not supported at the grid level, where we have an `@assert` everywhere, checking for cutoff degree to be less than polynomial degree component-wise. So having only one of the two, would require refactoring that.